_sort.py

```diff
@@ -0,0 +1,66 @@
+import os
+import yaml
+
+XT = "/etc/sysconfig/network-scripts/ifcfg-"
+SKIP = { "spaldingwcic0.chtc.wisc.edu.yaml", "unl-svc-1.facility.path-cc.io.yaml",
+         "path-ap2001.chtc.wisc.edu.yaml", "glidein-cm3000.chtc.wisc.edu.yaml"}
```

Comment:
Two thoughts:

1. For cleaner `git diff`s in the future, you should split this list to an item per line
2. Why are these specific nodes skipped? A comment explaining the list or each item in the list would be immensely helpful!